xlml/apis/task.py

```diff
@@ -92,11 +92,12 @@ def run_queued_resource_test(
           tpu_create_timeout,
           task_test_config,
       )
+
       queued_resource_op >> tpu.ssh_tpu.override(task_id="setup")(
           queued_resource_name,
           task_test_config.setup_script,
           ssh_keys,
-          all_workers,
+          True if task_test_config.test_name.startswith("tf_") else all_workers,
```

Comment:
It's kind of hard code here in `task.py`. Do we have any assertion to ensure tensorflow tests will always start with `tf` (I understand we manually format the name)? Otherwise, it will introduce a bug for future developers (as usually we don't check this logic when writing DAG).

Reply:
Yes, we will ensure all tf tests start with tf_ .

Reply:
How do you ensure this programmatically?